main/style.css

```diff
@@ -1,3 +1,8 @@
+/* Variables */
+:root {
```

Comment:
Why did you put the variable into root and not html.dark, where all the other theme-specific colors are specified?